Dockerfile

```diff
@@ -31,8 +31,11 @@ COPY --chown=appuser:appuser requirements.txt .
 RUN sed -i '/Items below this point will not be included in the Docker Image/,$d' requirements.txt && \
 	pip install --no-cache-dir --user -r requirements.txt
 
-# Copy the application files
-COPY --chown=appuser:appuser autogpt/ ./autogpt
+# Copy the entire repo to /app instead of to the home directory of appuser
+COPY --chown=appuser:appuser ./ /app
```

Comment:
This is covered in https://github.com/Significant-Gravitas/Auto-GPT/pull/1199

Reply:
Noted and I would prefer it that way.